<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client</class>
 <widget class="QWidget" name="client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>721</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="connectTab">
    <attribute name="title">
     <string>Connect</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_13">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>241</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Server Details</string>
     </property>
     <widget class="QLineEdit" name="ipfield">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>IP:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Port:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="portfield">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameField">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_14">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>201</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Status</string>
     </property>
     <widget class="QLabel" name="connectStatus">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Disconnected</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Duration: 15:02:22</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="connectButton">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="disconnectButton">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="downloadTab">
    <attribute name="title">
     <string>File Download</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>280</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Download Songs</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="downloadProgress">
     <property name="geometry">
      <rect>
       <x>102</x>
       <y>311</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QListWidget" name="downloadFileWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>221</width>
       <height>271</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Stressed Out - Twenty One Pilots</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sorry - Justin Bieber</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Renegades - X Ambassadors</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hide Away - Daya</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>See You Again - Wiz Khalifa</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>190</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Change Directory</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>220</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>C:/Downloads</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>311</y>
       <width>85</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Download Status:</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>201</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>Files</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of Songs:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Total Size:</string>
      </property>
     </widget>
    </widget>
    <widget class="QListWidget" name="listWidget_3">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>256</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Songs</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Downloaded Songs</string>
     </property>
    </widget>
    <widget class="QPushButton" name="updateSongButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>250</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Update Song List</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="uploadTab">
    <attribute name="title">
     <string>File Upload</string>
    </attribute>
    <widget class="QListWidget" name="listWidget_4">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>256</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Songs</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Uploaded Songs</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>190</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Change Directory</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>220</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>C:/Uploads</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>201</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>Files</string>
     </property>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of Songs:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Total Size:</string>
      </property>
     </widget>
    </widget>
    <widget class="QListWidget" name="listWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>221</width>
       <height>271</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Stressed Out - Twenty One Pilots</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sorry - Justin Bieber</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Renegades - X Ambassadors</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hide Away - Daya</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>See You Again - Wiz Khalifa</string>
      </property>
     </item>
    </widget>
    <widget class="QProgressBar" name="uploadProgress">
     <property name="geometry">
      <rect>
       <x>102</x>
       <y>311</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>311</y>
       <width>85</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Upload Status:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="uploadButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>280</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Upload Songs</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="chatTab">
    <attribute name="title">
     <string>Voice Chat</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>321</height>
      </rect>
     </property>
     <property name="title">
      <string>Online Users:</string>
     </property>
     <widget class="QListWidget" name="connectedWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>131</width>
        <height>281</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Alvin Man</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Colin Bose</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Martin Minkov</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Allen Tsang</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>End Chat</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Selected User:</string>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Martin Minkov</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>160</y>
       <width>201</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Status</string>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Connected</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Duration: 15:02:22</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>250</y>
       <width>201</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Call volume</string>
     </property>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>231</width>
       <height>321</height>
      </rect>
     </property>
     <property name="title">
      <string>Incoming Voice Chats</string>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>From: Alvin Man</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Accept</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Reject</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="voiceChatButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Call</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="streamTab">
    <attribute name="title">
     <string>Stream Music</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>431</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Stream Controls</string>
     </property>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Play/Pause</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Rewind</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>30</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Live</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>431</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Current Song</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>411</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Stitches - Shawn Mendes</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>251</width>
       <height>321</height>
      </rect>
     </property>
     <property name="title">
      <string>Playlist</string>
     </property>
     <widget class="QListWidget" name="listWidget_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>231</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_11">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>240</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Vote for Next Song</string>
     </property>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Selected Song: On My Mind - Ellie Goulding</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Vote</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_12">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>239</y>
       <width>191</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Music volume</string>
     </property>
     <widget class="QSlider" name="horizontalSlider_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>271</x>
       <y>170</y>
       <width>431</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>00:02</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>03:40</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
